api/docs/bt.dox

```diff
@@ -327,6 +327,19 @@ passes an \c instrlist_t of all Level 3 \c instr_t's, for simplicity.
   <tr><td><table width=100% bgcolor="#0000C0" cellspacing=0 cellpadding=1 border=0>
   <tr><td></td></tr></table></td></tr></table></td></tr></table>
 \endhtmlonly
+
+\if level_of_detail
+********************
+\subsection sec_IR_AArch64 Notes about the IR for AArch64
```

Comment:
s/Notes about the IR for AArch64/AArch64 NEON Vector Widths/

OK now I see the comment about OP_xx -- see my reply there.  If more is added here beyond NEON, I'm not sure what the title should be: "AArch64 IR Variations"?

Reply:
I think it's worth to update the docs and mention OP_xx here. It would probably be clearer to re-name it to OP_UNKNOWN, but ideally we would finish the decoder and get rid of it, to make it easier to spot holes in the decoder.